mapper/pojo/src/main/java/org/hibernate/search/mapper/pojo/model/spi/FieldPropertyHandle.java

```diff
@@ -12,10 +12,7 @@
 import org.hibernate.search.mapper.pojo.logging.impl.Log;
 import org.hibernate.search.util.common.logging.impl.LoggerFactory;
 
-/**
- * @author Yoann Rodiere
- */
-public final class FieldPropertyHandle implements PropertyHandle {
+final class FieldPropertyHandle implements PropertyHandle {
```

Comment:
A package private SPI?

Reply:
It's just implementation. It happens to only be needed in this package, so I kept it here and made it package private.